stratify/tests/test_vinterp.py

```diff
@@ -362,26 +382,51 @@ def test_result_dtype_f8(self):
 
 
 class Test__Interpolation_interpolate_z_target_nd(unittest.TestCase):
-    def test_target_z_3d_axis_0(self):
+    def test_target_z_3d_on_axis_0(self):
         z_target = z_source = f_source = np.arange(3) * np.ones([4, 2, 3])
         interp = vinterp._Interpolation(z_target, z_source, f_source,
                        axis=0, extrapolation=stratify.EXTRAPOLATE_NEAREST)
         result = interp.interpolate_z_target_nd()
         assert_array_equal(result, f_source)
 
-    def test_target_z_3d_axis_m1(self):
+    def test_target_z_3d_on_axis_m1(self):
         z_target = z_source = f_source = np.arange(3) * np.ones([4, 2, 3])
         interp = vinterp._Interpolation(z_target, z_source, f_source,
                        axis=-1, extrapolation=stratify.EXTRAPOLATE_NEAREST)
         result = interp.interpolate_z_target_nd()
         assert_array_equal(result, f_source)
 
+    def test_target_z_2d_over_3d_on_axis_1(self):
```

Comment:
Perhaps an illustrative comment would be useful here :)
Not 100% sure I'm following this test or the one below

Is this what's being tested?
```Python
_Interpolation(z_target(2, 4), z_src(3, 4), fz_src(3, 1, 1), axis=1))
```
Slightly confused sorry, axis is the fz_src axis to interpolate over, wouldn't that be 0??
(I appreciate it's not, I'm not quite following).

Cheers